components/usage/pkg/apiv1/usage_test.go

```diff
@@ -280,6 +287,7 @@ func TestReportGenerator_GenerateUsageReport(t *testing.T) {
 			ID:                 uuid.New(),
 			UsageAttributionID: db.NewTeamAttributionID(teamID.String()),
 			StartedTime:        db.NewVarcharTime(time.Date(2022, 06, 1, 1, 0, 0, 0, time.UTC)),
+			StoppingTime:       db.NewVarcharTime(time.Date(2022, 06, 1, 1, 0, 0, 0, time.UTC)),
 			StoppedTime:        db.NewVarcharTime(time.Date(2022, 06, 1, 1, 0, 0, 0, time.UTC)),
```

Comment:
the comment says: `No creation time, invalid record`. that should be no `startedTime` and `StartedTime` should be removed, I guess.

Reply:
Hmm, is this actually the case? This part of the query hasn't changed:

https://github.com/gitpod-io/gitpod/blob/135bc0f7586658b0e8730a2ff67d19bc7322b393/components/usage/pkg/db/workspace_instance.go#L79-L80

And when I remove the `StartedTime`, this changes the test behavior from {3 raw sessions; 1 invalid session; 2 usage records} to {2 raw sessions; 0 invalid session; 2 usage records}.

For now I've rolled back this suggestion, but please let me know if the test / the query logic should change here.